### DISCUSSION TITLE
Opt-out to background checks for places

### DISCUSSION THREAD

**Discussion by k0nstruct** [breezy-weather/breezy-weather#1241] [Ideas]
### Describe your idea

It would be nice to choose if a location could be disabled for background sync, I have many places registered to quickly check, but I don't need them to sync every hour:)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.6 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Did you read https://github.com/breezy-weather/breezy-weather/blob/main/docs/UPDATES.md#background-or-automatic-weather-update ?

**Reply by k0nstruct** [reply]
Oh, no, I forgot I had it to share info with my launcher. My bad.
